.papr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -xeuo pipefail
+
+export GOPATH=$HOME/gopath
+export PATH=$HOME/gopath/bin:$PATH
+export GOSRC=$HOME/gopath/src/github.com/projectatomic/libpod
+
+(mkdir -p $GOSRC && cd /code && cp -r . $GOSRC)
+
+dnf install -y \
+  bats \
+  btrfs-progs-devel \
+  bzip2 \
+  device-mapper-devel \
+  findutils \
+  git \
+  glib2-devel \
+  gnupg \
+  golang \
+  gpgme-devel \
+  libassuan-devel \
+  libseccomp-devel \
+  libselinux-devel \
+  make \
+  ostree-devel \
+  which
+
+# PAPR adds a merge commit, for testing, which fails the
+# short-commit-subject validation test, so tell git-validate.sh to only check
+# up to, but not including, the merge commit.
+export GITVALIDATE_TIP=$(cd $GOSRC; git log -2 --pretty='%H' | tail -n 1)
+export TAGS="seccomp $(hack/btrfs_tag.sh) $(hack/libdm_tag.sh) $(hack/btrfs_installed_tag.sh) $(hack/ostree_tag.sh) $(hack/selinux_tag.sh)"
```

Comment:
As an old-fuddy-duddy, I like having all my "constants" all at the beginning, so I always know where to find them.  I dunno though for this context, maybe that doesn't make sense.

Reply:
No problem moving but again this came from buildah.  We should probably be consistent.